app/controllers/results_controller.rb

```diff
@@ -5,6 +5,16 @@ def create
     render :ok, json: create_results
   end
 
+  def index
+    @results = Result.where(id: params[:id])
```

Comment:
> #create using the API key and #index using the 

In this PR?

> Also fetch the results using the project instance

Fetching using user instead (fetching a results shouldn't you require to specify the project)